change/@ni-nimble-components-99393253-07d1-4ec4-902b-41680bddd36e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
As an FYI, when the number of comments gets large it can be challenging to find open ones. On the files tab there is a dropdown of all the open comments to find them easier:

![image](https://github.com/ni/nimble/assets/1588923/6d15f31d-46d8-4f11-ade4-b554dfd2676c)